src/pages/NewChatConfirmPage.tsx

```diff
@@ -118,9 +119,9 @@ function NewChatConfirmPage() {
         // To handle this, we re-read the avatar image file from disk whenever the component re-mounts.
         readFileAsync(stashedLocalAvatarImage, newGroupDraft?.avatarFileName ?? '', onSuccess, onFailure, newGroupDraft?.avatarFileType ?? '');
 
-        // we only need to run this when the component re-mounted
+        // we only need to run this when the component re-mounted and when the onyx is loaded completely
```

Comment:
Minor change:

// we only need to run this when the component re-mounted and when onyx is loaded completely